sympy/matrices/matrices.py

```diff
@@ -4010,12 +4010,12 @@ def a2idx(j, n=None):
         try:
             j = j.__index__()
         except AttributeError:
-            raise IndexError("Invalid index a[%r]" % (j,))
+            raise IndexError("Invalid index a[%r]" % j)
```

Comment:
This was actually correct. The other way does the wrong thing if `j` is a tuple.